setup.cfg

```diff
@@ -1,50 +1,5 @@
-[metadata]
-name = pyraf
-version = 2.2.dev
-author = Rick White, Perry Greenfield, Chris Sontag
-author-email = help@stsci.edu
-home-page = http://www.stsci.edu/resources/software_hardware/pyraf
-summary = Provides a Pythonic interface to IRAF that can be used in place of the existing IRAF CL
-description-file = README
-requires-python = >=2.6
-classifier = 
-	Intended Audience :: Science/Research
-	License :: OSI Approved :: BSD License
-	Operating System :: OS Independent
-	Programming Language :: Python
-	Topic :: Scientific/Engineering :: Astronomy
-	Topic :: Software Development :: Libraries :: Python Modules
-requires-dist = 
-	stsci.tools
-
-[files]
-packages_root = lib
-packages = pyraf
-data_files = 
-	pyraf = data/blankcursor.xbm data/epar.optionDB data/pyraflogo_rgb_web.gif data/ipythonrc-pyraf
-	pyraf/clcache = data/clcache/*
-scripts = scripts/pyraf
-
-[extension=pyraf.sscanfmodule]
-sources = src/sscanfmodule.c
-optional = True
-fail_message = If this is Windows, it is ok.
-
-[extension=pyraf.xutilmodule]
-sources = src/xutil.c
-libraries = X11
-#optional = True
-#fail_message = If this is Windows, it is ok.
-
-[global]
-setup_hooks = 
-	stsci.distutils.hooks.use_packages_root
-	stsci.distutils.hooks.version_setup_hook
-	pyraf_setup.setup_hook
-commands = stsci.distutils.command.build_optional_ext.build_optional_ext
-
-[install_data]
-pre-hook.glob-data-files = stsci.distutils.hooks.glob_data_files
-
-[build_ext]
-pre-hook.pyraf-buildext-hook = pyraf_setup.build_ext_hook
+[bdist_wheel]
+# This flag says that the code is written to work on both Python 2 and Python
+# 3. If at all possible, it is good practice to do this. If you cannot, you
+# will need to generate wheels for each Python version that you support.
+universal=1
```

Comment:
This is not correct exactly.  The code is written to work on Py2, and to work on Py3 after 2to3 is run (with no hand editing).  Should we still have ```universal=1``` set?  I'm thinking not, but you know more about it.